packages/editor/src/components/post-actions/actions.js

```diff
@@ -612,6 +615,36 @@ const postRevisionsAction = {
 			onActionPerformed( posts );
 		}
 	},
+	context: 'view',
+};
+
+export const useEditPostAction = () => {
```

Comment:
In my mental model, the "editor" package (later probably core-data or a dedicated core-dataviews package) is about fields and actions that can work in any place.

This action has a direct dependency towards the site editor routes, so I believe it shouldn't be defined in the editor package and would prefer if it stays in the site editor (where the route is defined)